go/common/node/node.go

```diff
@@ -96,6 +96,9 @@ type Node struct { // nolint: maligned
 
 	// Roles is a bitmask representing the node roles.
 	Roles RolesMask `json:"roles"`
+
+	// SoftwareVersion is the node's oasis-node software version.
+	SoftwareVersion string `json:"software_version,omitempty"`
```

Comment:
Do we also need to bump the `LatestNodeDescriptorVersion` to the next version?

Reply:
Technically I should have done that when I added the VRF stuff, but, the storage node PR will do it.

Reply:
Ok, understood.